lib/tizen_plugins.dart

```diff
@@ -291,9 +291,6 @@ Future<void> ensureReadyForTizenTooling(FlutterProject project) async {
     return;
   }
   final TizenProject tizenProject = TizenProject.fromFlutter(project);
-  if (!tizenProject.existsSync()) {
-    return;
-  }
```

Comment:
Removing this check should be safe, because the check is also performed by `injectTizenPlugins`.